### ISSUE TITLE
User can modify the file order in diff view

### ISSUE THREAD

**Issue by damithc** [HubTurbo/HubTurbo#672]
... so that related files can be places together.

By default, GitHub orders them in alphabetical order, which may not be the optimal when reviewing a fix.

**Comment by acruis** [comment]
We don't have a diff view now, right?

**Comment by damithc** [comment]
yes, I meant the diff view in the bView. This is not urgent though.
